adobe_vipm/flows/utils/validation.py

```diff
@@ -1,4 +1,7 @@
-from adobe_vipm.flows.constants import PARAM_REQUIRED_CUSTOMER_ORDER, Param
+from adobe_vipm.flows.constants import (
+    PARAM_REQUIRED_CUSTOMER_ORDER,
+    Param,
+)
```

Comment:
I'm a bit confused with this change and the one on [line 63](https://github.com/softwareone-platform/swo-adobe-vipm-extension/pull/470/files#diff-0c923e4cd906b020dd1f11ef80237d9ab9e15b4e332ebbf0b4fcf71ff66147a7L63). Are they a Ruff change?